France-ioi/AlgoreaFrontend
# Feature/fix i18n texts

## Description

Fixes #902

## Notes (out of scope, known isues, hints for reviewing code, ...)  (optional)

...

## Test cases

  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/fix-i18n-texts/en/#/activities/by-id/1980584647557587953;path=4702,4102;attempId=0/edit/advanced-parameters)
  4. Then I see multiply (not implemented) captions

  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/en/#/activities/by-id/4702;path=;parentAttempId=0/details)
  4. Switch network to 3g and refresh browser.
  5. While loading turn network to offline
  6. Then I see "If this problem persists, please contact us." as before

  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/en/#/activities/by-id/1255005899678464217;path=39530140456452546,842524168135900505;attempId=0/details/progress/history)
  4. Switch network to offline and click on Content tab
  6. Then I see "An unknown error occurred. If this problem persists, please contact us."

  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/en/#/groups/by-id/672913018859223173;path=52767158366271444/details/members)
  4. Scroll down to Let user join using a code section
  6. Then hover on Validity: each buttons 
  7. Then I see tooltip with part of text (reset current expiration)
  8. Click on Custom button
  9. Hover on checked button
  10. Then I see tooltip with part of text (reset current expiration)

- [ ] Case 5:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/en/#/activities/by-id/39530140456452546;path=4702,4102,1980584647557587953;attempId=0/edit/advanced-parameters)
  4. Scroll down to bottom 
  6. Then I see "Only empty chapters can be deleted."

- [ ] Case 6:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/en/#/skills/by-id/7164018311292249641;path=3000,3002,2518359812137710892;attempId=0/edit/advanced-parameters)
  4. Scroll down to bottom 
  6. Then I see "Only empty skills can be deleted."
